src/Symfony/Bridge/Monolog/composer.json

```diff
@@ -23,6 +23,7 @@
     "require-dev": {
         "symfony/console": "~2.8|~3.0|~4.0",
         "symfony/event-dispatcher": "~2.8|~3.0|~4.0",
+        "symfony/security-core": "~2.8|~3.0",
```

Comment:
then I suggest to remove the type int, it's purely internal and does justify a deps limitation